BuildScript/BuildTargets/Testing.targets

```diff
@@ -127,16 +119,31 @@
       <_mergedTestCategoriesToExclude>$(TestCategoriesToExclude.Replace(';', ',')),$(_excludeCategoriesInTestConfiguration)</_mergedTestCategoriesToExclude>
       <_mergedTestCategoriesToExclude>$(_mergedTestCategoriesToExclude.Trim(','))</_mergedTestCategoriesToExclude>
 
-      <_excludeCategoriesCallArgumentString></_excludeCategoriesCallArgumentString>
-      <_excludeCategoriesCallArgumentString Condition="'$(_mergedTestCategoriesToExclude)' != ''">/exclude=$(_mergedTestCategoriesToExclude)</_excludeCategoriesCallArgumentString>
+      <_mergedTestCategoriesToInclude>$(TestCategoriesToInclude.Replace(';', ',')),$(_includeCategoriesInTestConfiguration)</_mergedTestCategoriesToInclude>
+      <_mergedTestCategoriesToInclude>$(_mergedTestCategoriesToInclude.Trim(','))</_mergedTestCategoriesToInclude>
     </PropertyGroup>
 
+    <ItemGroup>
+      <SplitCategoriesToExclude Include="$(_mergedTestCategoriesToExclude.Split(','))"/>
```

Comment:
is those private state or shared between targets. if private, please use _lowerCase syntax. Also, please clear the itemgroups before usage (see other code-places)